docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -348,6 +348,310 @@ and a file system location of ``/var/my_tables/test_table``::
         partitioning = ARRAY['c1', 'c2'],
         location = '/var/my_tables/test_table')
 
+.. _iceberg-special-tables:
+
+Special tables
+--------------
+
+The Iceberg connector maintains several hidden tables that provide metadata
+for a specific table. You can query each metadata table by appending the
+metadata table name to the table name::
+
+   SELECT * FROM "test_table$data"
+
+``$data`` table
+^^^^^^^^^^^^^^^
+
+The ``$data`` table is an alias for the Iceberg table.
+
+The statement::
+
+    SELECT * FROM "test_table$data"
+
+is equivalent to::
+
+    SELECT * FROM test_table
+
+``$properties`` table
+^^^^^^^^^^^^^^^^^^^^^
+
+The ``$properties`` table provides general metadata information about the table.
+
+You can retrieve the changelog of the Iceberg table ``test_table``
+by using the query::
```

Comment:
Question: This is likely outside the scope of this PR, but since this is a changelog of the table properties, are there any additional plans to provide information on current info or when something was added?

I know Iceberg doesn't natively provide this functionality, so it might be difficult but wondering how the changelog is provided.

Reply:
Could you please provide an example so that I understand what you mean?
Here is the implementation of the functionality behind the `$properties` table
https://github.com/trinodb/trino/blob/master/plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/PropertiesTable.java

Reply:
`changelog` term was falsely placed.  sorry for the confusion.